src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -28,7 +28,7 @@
 SSLERROR_TEMPLATE = ('Certificate verification failed. This typically happens when using Azure CLI behind a proxy '
                      'that intercepts traffic with a self-signed certificate. '
                      # pylint: disable=line-too-long
-                     'Please add this certificate to the trusted CA bundle: https://github.com/Azure/azure-cli/blob/dev/doc/use_cli_effectively.md#work-behind-a-proxy. '
+                     'Please add this certificate to the trusted CA bundle: https://docs.microsoft.com/en-us/cli/azure/use-cli-effectively#work-behind-a-proxy. '
```

Comment:
We still lack a good way to manage short links. Perhaps we need a meeting to discuss about how to manage them.